flocker/acceptance/test_api.py

```diff
@@ -404,6 +442,24 @@ def restarted(record):
         creating_dataset.addCallback(created_dataset)
         return creating_dataset
 
+    def test_current(self):
+        """
+        The current container endpoint includes a currently running container.
+        """
+        creating = self._create_container()
+
+        def created(result):
+            cluster, data = result
+            data[u"running"] = True
```

Comment:
The API docs say that `/state/containers` endpoint will return running and non-running containers.

Should there be a separate test for listing a container that is not running?